.travis.yml

```diff
@@ -4,12 +4,12 @@ sudo: false
 
 python:
   - "2.7"
+  - "3.3"
   - "3.4"
   - "3.5"
   - "3.6"
   - "pypy"
-  - "2.6"
-  - "3.3"
+  # - "2.6"
```

Comment:
I'd be fine with dropping Python2.6 support

@jstasiak I'm afraid that you've been the only one that was using it on that version, is that still the case?

Reply:
@berdario That was three years ago, today I'm of a position that it's fine to drop support for anything older than 3.4 ;)